components/data/data-services/org.wso2.micro.integrator.dataservices.core/src/test/java/org/wso2/micro/integrator/dataservices/core/test/util/TestUtils.java

```diff
@@ -98,7 +103,34 @@ public static OMElement callOperation(String epr, String opName,
 		return result;
 	}
 
-        /**
+	public static OMElement callOperation2(String epr, String opName,
+										  Map<String, String> params) throws AxisFault {
+		EndpointReference targetEPR = new EndpointReference(epr);
+		OMElement payload = getPayload(opName, params);
+		Options options = new Options();
+		options.setTo(targetEPR);
+		options.setAction("urn:" + opName);
+		ServiceClient sender = new ServiceClient();
+		sender.setOptions(options);
+		OMElement result = sender.sendReceive(payload);
+		return result;
+	}
+
+	public static OMElement callOperation3(String epr, String opName,
+										   Map<String, String> params) throws AxisFault {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line is longer than 120 characters (found 129).](https://app.codacy.com/app/sdkottegoda/micro-integrator/pullRequest?prid=4575789)